### PR TITLE
Add instructions to run the hybrid simulation

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🎉 New feature

## Summary

Add instructions to run the hybrid simulation

## Test it

Follow this [README.md](https://github.com/ignitionrobotics/ign-omni/blob/ahcorde/hybrid_instructions/tutorials/hybrid_simulation.md)

Remember to checkout this branch in the ign-omni repository.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by koonpeng — COMMENTED** [review]
Ignition crashes when I start the simulation. I'm using galactic+ediface, ignition is using the system instance, the `ign-omni` workspace is not sourced.

edit: issue was due to some other versions of ignition installed and it's linking to the wrong version.

**Comment by koonpeng** [comment]
After many `Multiple texture coordinate sets exist in submesh` warnings, the connector crashes with
```
[Err] [SystemPaths.cc:467] Could not resolve file [FANS_Albedo.png]
[Err] [SystemPaths.cc:467] Could not resolve file [FANS_Metalness.png]
[Err] [SystemPaths.cc:467] Could not resolve file [FANS_Normal.png]
[Err] [SystemPaths.cc:467] Could not resolve file [FANS_Roughness.png]
[Dbg] [Scene.cpp:649] added model [Depot]
[Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[Wrn] [ColladaLoader.cc:2394] Triangle input semantic: 'COLOR' is currently not supported
[Err] [Scene.cpp:753] This kind of sensor [] is not supported
[Err] [Scene.cpp:381] Failed to add sensor [/depot/standard_dock/green_buoy_link/green_buoy]
[Err] [Scene.cpp:622] Failed to update model [standard_dock]
[Err] [Scene.cpp:646] Failed to add model [standard_dock]
[Err] [Scene.cpp:840] Failed to init scene
```

**Comment by koonpeng** [comment]
> Here you can find a fix https://github.com/ignitionrobotics/ign-omni/pull/22. Do you mind to try again?

The connector is able to run now. But I'm still not able to get the lidar scans.

This part in the instructions:
```
Configure the Plugin:

The frame_id should be turtlebot4/rplidar_link/rplidar
The laser scan topic /scan_issac
The lidar prim /depot/turtlebot4/rplidar_link/rplidar
```

I don't have the lidar prim `/depot/turtlebot4/rplidar_link/rplidar`.

image

I also don't see anything in rviz.

image

**Comment by ahcorde** [comment]
do you mind to try with this branch `ahcorde/issacsim_2_ignition` ?

**Comment by koonpeng** [comment]
> do you mind to try with this branch ahcorde/issacsim_2_ignition ?

Same results, but i notice that when i listen to `/scan_ignition` rviz reports lots of such errors

```
[rviz2-15] [INFO] [1648190813.067043341] [rviz2]: Message Filter dropping message: frame 'turtlebot4/rplidar_link/rplidar' at time 109.616 for reason 'discarding message because the queue is full'
```

There are no messages when listen on `/scan` and `/scan_issac`.

Other than that, the simulation seems to work, I can move the robot etc.

File: tutorials/hybrid_simulation.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+    def __init__(self):
+        super().__init__('minimal_subscriber')
+        self.subscription = self.create_subscription(
+            LaserScan,
+            '/scan_issac',
+            self.listener_callback,
+            10)
+        self.subscription  # prevent unused variable warning
+        self.publisher_ = self.create_publisher(LaserScan, '/scan', 10)
```
Comment (koonpeng) [inline tutorials/hybrid_simulation.md:138]: Should this be listening on `/scan` and republishing to `/scan_issac` instead? I tried changing this but still can't get the sensors to show up.

**Review by adlarkin — COMMENTED** [review]
Here are a few comments that I have about the documentation I went through (tutorials 1 and 2):

---

After running the last command in https://github.com/ignitionrobotics/ign-omni/blob/ahcorde/hybrid_instructions/tutorials/01_compile.md (`colcon build --merge-install --event-handlers console_direct+ --packages-up-to ignition-omniverse1`), I see the following warning (the build seemed to succeed, though):
```
WARNING:colcon.colcon_cmake.task.cmake.build:Could not run installation step for package 'ignition-omniverse1' because it has no 'install' target
```

Is this something to worry about, or can it be ignored? If it can be ignored, I think it would be worth mentioning this in the documentation.

---

In the [`Run Ignition` section](https://github.com/ignitionrobotics/ign-omni/blob/ahcorde/hybrid_instructions/tutorials/02_quickstart.md#run-ignition) of tutorial 2 (quickstart), I think the following command can be removed because Fortress binaries were installed in tutorial 1:
```
source ~/ignition-fortress/install/setup.bash
```

---

In the [`Run the connector` section](https://github.com/ignitionrobotics/ign-omni/blob/ahcorde/hybrid_instructions/tutorials/02_quickstart.md#run-the-connector) of tutorial 2 (quickstart), it looks like a command is missing (note how there's nothing after `And finally run:`:

missing_cmd

---

In the [`Run IsaacSim` section](https://github.com/ignitionrobotics/ign-omni/blob/ahcorde/hybrid_instructions/tutorials/02_quickstart.md#run-isaacsim) of tutorial 2 (quickstart), there's an instruction to `Launch IsaacSim and activate the live sync`. The gif below this line already has `IsaacSim` running, but can instructions be shown on how to start `IsaacSim` if it's not running already?

**Pull request merged**